cms/schemas/blockContent.js

```diff
@@ -48,6 +50,25 @@ export default {
                 type: "url",
               },
             ],
+            blockEditor: {
+              icon: () => <div>🔗</div>,
+            },
+          },
+          {
+            title: "Internal link",
+            name: "internalLink",
+            type: "object",
+            fields: [
+              {
+                name: "item",
+                type: "reference",
+                to: [{ type: "fileUpload" }],
+                validation: (Rule) => Rule.required(),
+              },
+            ],
+            blockEditor: {
+              icon: () => <div>🗂</div>,
```

Comment:
looks like a weird black blob on my windows  screen, can we try this emoji instead:
```suggestion
              icon: () => <div>📂</div>,
```

Reply:
![image](https://user-images.githubusercontent.com/88635679/157115134-daaa863a-ae4d-4254-9f8c-bf6308e1d12c.png)